tools/topology/topology2/platform/intel/dmic-default.conf

```diff
@@ -47,4 +47,10 @@ Define {
 
 	# Note: This will be redefined in dmic-generic.conf if not set from cmake
 	DMIC0_PCM_CHANNELS		0
+
+	# Deep buffer capture
+	DMIC0_DEEP_BUFFER_PCM_NAME	"DMIC Deep Buffer"
+	DMIC0_DEEP_BUFFER_PIPELINE_ID	18
+	DMIC0_DEEP_BUFFER_PCM_ID	46
```

Comment:
Longterm, do we have a topology file that lists all PCM IDs ? Not a blocker now.